### PR TITLE
fix: alignment of user actions on posts

### LINKED ISSUE #3214: UI issue with card grid view only on tabs
### Context

If you have a gride view with full controls plus reorder controls the delete button will be outside the card view 
this issue happens only on tabs not in the main resource index view

### System configuration
**Avo version**: 3.12.0

**Rails version**: 7.2

**Ruby version**: 3.3.4

**License type**

 - [ ] Community
 - [ ] Pro
 - [x] Advanced

**Are you using Avo monkey patches, overriding views or view components?**

 - [ ] Yes. If so, please post code samples.
 - [x] No

### Screenshots or screen recordings
IMG-20240903-WA0002

### PR DESCRIPTION
**Pull request by Aaryanpal**
# Description

Fixes #3214

# Checklist:

- [x] I have performed a self-review of my own code

## Screenshots & recording

- [x] ScreenCast Attatched

https://github.com/user-attachments/assets/3f8d344a-af83-466d-920a-c2d549b705a5

### CODE REVIEW

File: app/components/avo/index/resource_controls_component.html.erb [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<%= content_tag :div, class: "space-x-2 flex flex-row justify-between ml-2 has-record-selector:ml-0" do %>
+<%= content_tag :div, class: "#{params[:view_type] == "grid" ? "flex-wrap" : "space-x-2"} flex flex-row justify-evenly ml-2 has-record-selector:ml-0" do %>
```
Comment (adrianthedev) [inline app/components/avo/index/resource_controls_component.html.erb:1]:
```suggestion
<%= content_tag :div, class: "#{view_type == "grid" ? "flex-wrap" : "space-x-2"} flex flex-row justify-evenly ml-2 has-record-selector:ml-0" do %>
```

Can we please extract `view_type` inside the component?

**Review by adrianthedev — CHANGES_REQUESTED** [review]
I left a small piece of feedback. Thank you!

File: app/components/avo/index/resource_controls_component.html.erb (hunk 1 above)
Comment (Aaryanpal) [reply app/components/avo/index/resource_controls_component.html.erb]: Done! @adrianthedev

**Review by Paul-Bob — APPROVED** [review]
Thank you @Aaryanpal!

**Pull request merged**